cardano-api/src/Cardano/Api/LocalStateQuery.hs

```diff
@@ -0,0 +1,188 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Cardano.Api.LocalStateQuery
+  ( queryPParamsFromLocalState
+  ) where
+
+import           Cardano.Prelude hiding (ByteString, atomically, option, threadDelay)
+
+import           Cardano.Api.TxSubmit.Types (textShow)
+
+import           Cardano.BM.Data.Tracer (ToLogObject (..), nullTracer)
+import           Cardano.BM.Trace (Trace, appendName, logInfo)
+
+import           Cardano.Config.Protocol ()
+import           Cardano.Config.Types (SocketPath (..))
+
+import qualified Codec.CBOR.Term as CBOR
+
+import           Control.Monad.Class.MonadSTM.Strict
+                   (MonadSTM, StrictTMVar, atomically, newEmptyTMVarM, tryPutTMVar, takeTMVar)
+import           Control.Monad.Trans.Except.Extra (newExceptT)
+import           Control.Tracer (Tracer)
+
+import           Data.ByteString.Lazy (ByteString)
```

Comment:
I think `Cardano.Prelude` has a `LByteString` which means you would not have to hide it and import it here.